HotelYeah/package.json

```diff
@@ -18,6 +18,7 @@
     "axios": "^0.21.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-native": "^0.63.4",
```

Comment:
is there a reason this dep has been added? i thought react-native was for android/iOS apps

Reply:
no reason for this one, my bad, I forgot to remove it...